07-errors.md

```diff
@@ -0,0 +1,301 @@
+# Error Conditions
+
+This document specifies the error conditions in the FVM and the associated error codes/numbers.
+
+## Exit Codes
+
+Exit codes are numeric, 32bit integer codes that are returned when an actor exits both successfully and in error.
+Exit codes appear on-chain inside the message receipt, and therefore form part of consensus.
```

Comment:
I agree with everything but:

> In both cases, non-zero exit codes interrupt the execution flow.

Because the events happen in the other order: execution is interrupted, then we exit with a non-zero exit code.

How about:

> In both cases, a non-zero exit code indicates that execution was interrupted and that the actors state was left unchanged.

Reply:
LGTM.